addon/ng2/blueprints/ng2/files/config/karma.conf.js

```diff
@@ -14,31 +14,17 @@ module.exports = function (config) {
       }
     },
     files: [
-      { pattern: 'node_modules/systemjs/dist/system-polyfills.js', included: true, watched: true },
-      { pattern: 'node_modules/systemjs/dist/system.src.js', included: true, watched: true },
-      { pattern: 'node_modules/es6-shim/es6-shim.js', included: true, watched: true },
-      { pattern: 'node_modules/angular2/bundles/angular2-polyfills.js', included: true, watched: true },
-      { pattern: 'node_modules/zone.js/dist/async-test.js', included: true, watched: true },
-      { pattern: 'node_modules/zone.js/dist/fake-async-test.js', included: true, watched: true },
-      { pattern: 'node_modules/rxjs/bundles/Rx.js', included: true, watched: true },
-      { pattern: 'node_modules/angular2/bundles/angular2.js', included: true, watched: true },
-      { pattern: 'node_modules/angular2/bundles/http.dev.js', included: true, watched: true },
-      { pattern: 'node_modules/angular2/bundles/router.dev.js', included: true, watched: true },
-      { pattern: 'node_modules/angular2/bundles/testing.dev.js', included: true, watched: true },
+      { pattern: 'dist/vendor/es6-shim/es6-shim.js', included: true, watched: false },
+      { pattern: 'dist/vendor/zone.js/dist/zone.js', included: true, watched: false },
+      { pattern: 'dist/vendor/reflect-metadata/Reflect.js', included: true, watched: false },
+      { pattern: 'dist/vendor/systemjs/dist/system-polyfills.js', included: true, watched: false },
+      { pattern: 'dist/vendor/systemjs/dist/system.src.js', included: true, watched: false },
+      { pattern: 'dist/vendor/zone.js/dist/async-test.js', included: true, watched: false },
 
       { pattern: 'config/karma-test-shim.js', included: true, watched: true },
 
-      // paths loaded via module imports
-      { pattern: 'dist/**/*.js', included: false, watched: true },
-
-      // paths loaded via Angular's component compiler
-      // (these paths need to be rewritten, see proxies section)
-      { pattern: 'dist/**/*.html', included: false, watched: true },
-      { pattern: 'dist/**/*.css', included: false, watched: true },
-
-      // paths to support debugging with source maps in dev tools
-      { pattern: 'dist/**/*.ts', included: false, watched: false },
-      { pattern: 'dist/**/*.js.map', included: false, watched: false }
+      // Distribution folder.
+      { pattern: 'dist/**/*', included: false, watched: true }
     ],
     proxies: {
```

Comment:
With SystemJS being correctly configured, we can make away with the proxy entry here.

I removed it in https://github.com/angular/angular-cli/pull/481 and that fixed a warning we also got, and also an infinite proxy looped that happened sometimes.